src/helpers.ts

```diff
@@ -0,0 +1,230 @@
+/*!
```

Comment:
While you're in here, can we update this license header to be consistent with the generated ones?  (keeping the 2017 year of course)

Reply:
sure, done!